packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -11,20 +11,14 @@ import {
   WebpackLoggingCallback,
```

Comment:
From what I am seeing localization only happens if DL is enabled. But what if users don't have DL enabled?

Reply:
Current limitation of initial support.  Follow up PRs will expand support.